catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/ValidateCommand.java

```diff
@@ -47,178 +55,104 @@
 @Service
 public class ValidateCommand implements Action {
 
-    @Argument(index = 0, name = "fileName", description = "The path to the file that you want to validate", required = true, multiValued = false)
+    @Option(name = "--path", aliases = "-p", description = "The path to the file to be validated")
```

Comment:
@coyotesqrl did you have any recommendations on how to specify between whether the user is attempting to provide a path or a query without using `@Option`? I see that `karaf.shell.commands.option` is marked as deprecated, but it doesn't appear that `karaf.shell.api.action.option` is.